### ISSUE TITLE
Don't I need to retrain after validating TFT model?

### ISSUE THREAD

**Issue by iba13001** [jdb78/pytorch-forecasting#1367]
I saw in the tutorial that after validation, prediction was directly made on a new set without including the validation data in the training.
Why's that happening?
Isn't that was, we are losing the information in the validation set?
